app/views/users/index.html.erb

```diff
@@ -1,11 +1,13 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>All Users</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-  </head>
+<h1>List of all Users with their number of posts only</h1>
 
-  <body>
-    <h1> List of all Users with their number of posts only </h1>
-  </body>
-</html>
\ No newline at end of file
+<% @users.each_with_index do |user, index| %>
```

Comment:
- [ ] [OPTIONAL] Kindly add navigation links to all different pages, make a button to navigate to create posts and create comment pages, and add a button to add like for posts for a better user experience.